Java/src/main/java/epam/Planes/Plane.java

```diff
@@ -0,0 +1,55 @@
+package epam.Planes;
+
+import java.util.Objects;
+
+
+abstract public class Plane {
+	String model;
+	private int maxSpeed;
+	private int maxFlightDistance;
+	private int maxLoadCapacity;
+
+	public Plane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity) {
+		this.model = model;
+		this.maxSpeed = maxSpeed;
+		this.maxFlightDistance = maxFlightDistance;
+		this.maxLoadCapacity = maxLoadCapacity;
+	}
+
+	public int getMaxSpeed() {
+		return maxSpeed;
+	}
+
+	public int getMaxFlightDistance() {
+		return maxFlightDistance;
+	}
+
+	public int getMaxLoadCapacity() {
+		int result = this.maxLoadCapacity;
+		return result;
+	}
+
+	@Override
+	public String toString() {
+		return "Plane{" + "model='" + model + '\'' + ", maxSpeed=" + maxSpeed + ", maxFlightDistance=" + maxFlightDistance
+				+ ", maxLoadCapacity=" + maxLoadCapacity + '}';
+	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
change the name of variable "o", variable names must be self descriptive

Reply:
Done